ligo/Makefile

```diff
@@ -22,7 +22,7 @@ OUT ?= out
 
 .PHONY: all clean test
 
-all: $(OUT)/baseDAO.tz $(OUT)/registryDAO_storage.tz
+all: $(OUT)/baseDAO.tz $(OUT)/registryDAO_storage.tz $(OUT)/treasuryDAO_storage.tz
```

Comment:
As we did for registryDAO, now that we have a `make` way to create this, I think it should be used in `ligo/README.md` instead of `compile-storage` directly.

Reply:
I've updated this now.